Company API Flow/Adverse Action of Single Employee.md

```diff
@@ -0,0 +1,38 @@
+# cURL
+
```

Comment:
This file should be inside /source/includes folder.
Please Refer this link: https://github.com/slatedocs/slate/issues/345#issuecomment-456634999

Once Done run : bundle exec middleman build --clean
and test it locally